src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImplTest.java

```diff
@@ -87,7 +86,6 @@ void processMessageTest() {
 
     @Test
     void processErrorMessageTest() {
-        ReflectionTestUtils.setField(messageProcessorService, "runAppInErrorMode", true);
```

Comment:
I think this test is now the same as "test retry is called" so we can probably remove this test entirely

Reply:
i'll take a look